src/commands/flow/internal/executeFlow.ts

```diff
@@ -0,0 +1,213 @@
+import SASjs, { PollOptions } from '@sasjs/adapter/node'
+import { AuthConfig, Target } from '@sasjs/utils'
+import { Command } from '../../../utils/command'
+import { displayError, displaySuccess } from '../../../utils/displayResult'
+import { millisecondsToDdHhMmSs } from '../../../utils/utils'
+import {
+  saveLog,
+  saveToCsv,
+  normalizeFilePath,
+  parseJobDetails,
+  printError,
+  generateFileName
+} from '.'
+import csvColumns from './csvColumns'
+import { FlowWave, FlowWaveJob } from '../../../types'
+
+export const executeFlow = async (
+  flow: FlowWave,
+  sasjs: SASjs,
+  pollOptions: PollOptions,
+  target: Target,
+  authConfig: AuthConfig,
+  csvFile: string
+): Promise<{
+  jobStatus: boolean
```

Comment:
would be nice to have a type for returned value

Reply:
added `executeFlowReturn`